pkg/utils/file.go

```diff
@@ -0,0 +1,19 @@
+package utils
+
+import (
+	"go.uber.org/zap"
+	"os"
+)
+
+// GetFileMode fetches the filemode from a file path
+func GetFileMode(path string) os.FileMode {
+	fi, err := os.Stat(path)
+	if err != nil {
+		if os.IsNotExist(err) {
+			zap.S().Errorf("file %s does not exist.", path)
+		} else {
+			zap.S().Errorf("unable to fetch file info for path %s.", path)
+		}
```

Comment:
This method may panic. os.Stat() returns nil file info on an error and we are just logging the error and not returning.